tests/sdl_key_proxy.c

```diff
@@ -61,9 +61,7 @@ int main(int argc, char **argv) {
   printf("main\n");
   SDL_Init(SDL_INIT_VIDEO);
   SDL_Surface *screen = SDL_SetVideoMode(600, 450, 32, SDL_HWSURFACE);
-
-  if (argc == 1337) one(); // keep it alive
```

Comment:
Several of these tested used this method to keep the `one()` symbol from being DCE'd but using `EMSCRIPTEN_KEEPALIVE` is a more explicit way to do this.

These test call the `one` function from JS.  For example the test_sdl_text test that you commented on does this in JS to make the test work:

```
      Module.postRun = function() {
        function doOne() {
          Module._one();
          setTimeout(doOne, 1000/60);
        }
        setTimeout(doOne, 1000/60);
      }
```

Its not clear to me why the test doesn't just use the native timeout APIs that emscripten provides, but thats a different matter.